### PR TITLE
Added pages

### PR DESCRIPTION
**Pull request by nishasen**
Added Pages
1. Home
2. Explore
3. Profile
4. bookmark
5. notification
6. settings
7. user
8. error

### CODE REVIEW

File: social/src/Pages/Landing/Landing.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Hero from '../../Assets/Hero.svg';
+import { LoginForm } from '../../Components';
+import '../Styles.css';
+// import { useDispatch } from 'react-redux';
+// import { loggedOut } from '../../Features';
```
Comment (sadathshariff) [inline social/src/Pages/Landing/Landing.jsx:6]: Remove Console logs

File: social/src/Pages/Landing/Landing.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+const Landing = () => {
+  // const dispatch = useDispatch();
+  // dispatch(loggedOut());
```
Comment (sadathshariff) [inline social/src/Pages/Landing/Landing.jsx:10]: here as well

**Review by sadathshariff — APPROVED** [review]
Good to go, Everything looks fine

**Review by srejitk — APPROVED** [review]
Good work handing the user profile section. Do remove the unnecessary logs.
Looks good otherwise.

File: social/src/Pages/Landing/Landing.jsx (hunk 1 above)
Comment (nishasen) [reply social/src/Pages/Landing/Landing.jsx:6]: Yes will do this

File: social/src/Pages/Landing/Landing.jsx (hunk 2 above)
Comment (nishasen) [reply social/src/Pages/Landing/Landing.jsx:10]: Okay

**Pull request merged**